jobmon_server/src/jobmon/server/web/models/task_instance.py

```diff
@@ -166,6 +166,8 @@ def to_wire_as_worker_node_task_instance(self) -> Tuple:
         (TaskInstanceStatus.TRIAGING, TaskInstanceStatus.ERROR_FATAL),
         # task instance error after transitioning from kill_self
         (TaskInstanceStatus.KILL_SELF, TaskInstanceStatus.ERROR_FATAL),
+        # task is reset by workflow resume and worker finishes gracefully
```

Comment:
Just confirming for my own understanding - for example this transition could happen if a user stops their workflow, but while Jobmon is shutting it down/cleaning it up one of the TIs successfully completes in that time, correct? Is this more for edge case handling?